python/nano/src/bigdl/nano/deps/openvino/pytorch/pytorch_openvino_model.py

```diff
@@ -76,6 +78,21 @@ def _load(path):
         xml_path = Path(path) / status['xml_path']
         return PytorchOpenVINOModel(xml_path)
 
-    def _save_model(self, path):
-        xml_path = Path(path) / self.status['xml_path']
-        super()._save_model(xml_path)
+    def pot(self,
+            dataloader,
+            metric=None,
+            higher_better=True,
+            drop_type="relative",
+            maximal_drop=0.999,
+            max_iter_num=1,
+            n_requests=None,
+            sample_size=300):
+        # convert torch metric/dataloader to openvino format
+        if metric:
+            metric = PytorchOpenVINOMetric(metric=metric, higher_better=higher_better)
+        dataloader = PytorchOpenVINODataLoader(dataloader, collate_fn=self.tensors_to_numpy)
```

Comment:
need to check the type of metric & dataloader somewhere?

Reply:
added checks in `PytorchOpenVINOMetric `and `PytorchOpenVINODataLoader`